docs/reference/setup/rolling_upgrade.asciidoc

```diff
@@ -87,7 +87,26 @@ To upgrade using a zip or compressed tarball:
     to your new installation, or configure the location of the data directory
     in the `config/elasticsearch.yml` file, with the `path.data` setting.
 
-==== Step 4: Start the upgraded node
+==== Step 4: Upgrade Plugins
+
+Now that the elasticsearch packages have been updated, you should update any plugins you have installed as well. Please make sure you check the release notes for any plugins before installing them. To update the plugins:
+
+*   First, remove the exising plugin with `bin/plugin remove <plugin name>`.
```

Comment:
I will suggest to first get all the installed plugins:

```
➜  elasticsearch-2.1.0  bin/plugin list
Installed plugins in /Users/Gabriel/Documents/ElasticSearch/elasticsearch-2.1.0/plugins:
    - license
    - marvel-agent
    - shield
```

Then, for each plugin listed in the list output, do all this steps.